### PR TITLE
ENH: Allow float decim as dest

### PR DESCRIPTION
**Pull request by larsoner**
Useful for @ktavabi who has data sampled at 1800 and 1200 Hz, allowing `decim=300.` is a lot nicer than having to pass a list of decim values. @nordme @mdclarke can you look as well, as it might be relevant to you, too? Some of your data might be this way, not sure.

### CODE REVIEW

**Comment by mdclarke** [comment]
This works for me. Thanks for this, it's helpful for a lot of datasets :)

**Comment by larsoner** [comment]
Okay great, in it goes! Others still feel free to look and comment

File: mnefun/_utils.py [hunk 1]
```diff
@@ -80,6 +80,19 @@ def _handle_dict(entry, subj):
     return out
 
 
+def _handle_decim(decim, sfreq):
+    decim = np.array(decim)
+    assert decim.shape == ()
```
Comment (ktavabi) [inline mnefun/_utils.py:85]: I assume you tested and this works, but what's the reasoning for this assertion? Isn't always going to be a 'tuple'..is it like a stealthy `type` check?

File: mnefun/_utils.py [hunk 2]
```diff
@@ -80,6 +80,19 @@ def _handle_dict(entry, subj):
...
+    if decim.dtype.char in 'il':
```
Comment (ktavabi) [inline mnefun/_utils.py:86]: Again, for my own personal edification: what is `il`?

File: mnefun/_utils.py (hunk 1 above)
Comment (larsoner) [reply mnefun/_utils.py:85]: `np.array(..)` can return arbitrary shapes, this ensures it's a scalar as:
```
>>> np.array(1.).shape
()
>>> np.array([1.]).shape
(1,)
```
etc.

File: mnefun/_utils.py (hunk 2 above)
Comment (larsoner) [reply mnefun/_utils.py:86]: int or long

Comment (ktavabi) [reply mnefun/_utils.py:86]: Maybe comment w/ _NumPy dtype attribute for workspace names_ if that's the correct terminology. That's some Kung Fu Panda shit :)

**Comment by ktavabi** [comment]
Feel free to close.

**Pull request merged**